spring-cloud-stream-binder-kinesis/src/test/java/org/springframework/cloud/stream/binder/kinesis/LocalKinesisResource.java

```diff
@@ -53,7 +49,6 @@ protected void obtainResource() throws Exception {
 		System.setProperty(SDKGlobalConfiguration.AWS_CBOR_DISABLE_SYSTEM_PROPERTY, "true");
 
 		this.resource = AmazonKinesisAsyncClientBuilder.standard()
-				.withCredentials(mock(AWSCredentialsProvider.class))
```

Comment:
Why did you removed this?
How does it work then?

I didn't check how the Kinesalite works because I'm on Windows 😄 , but I don't believe that we can stay with something default for credentials.